docs/package.json

```diff
@@ -36,15 +35,12 @@
     "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^9.0.0",
     "babel-loader": "^8.1.0",
```

Comment:
cc @cerinman @phantomxc I am pretty sure we don't need jest or enzyme in the docs project because there are no tests there.